src/Compilers/CSharp/Test/Emit2/CodeGen/CodeGenSpanBasedStringConcatTests.cs

```diff
@@ -586,15 +586,40 @@ static void Main()
         var comp = CreateCompilation(source, options: TestOptions.ReleaseExe, targetFramework: TargetFramework.Net80);
         comp.MakeMemberMissing(SpecialMember.System_Object__ToString);
 
-        // Although we don't use object.ToString() or char.ToString() in the final codegen we still need object.ToString() during lowering.
-        // Moreover, we previously reported these errors anyway, so this is not a behavioral change
-        comp.VerifyEmitDiagnostics(
-            // (13,47): error CS0656: Missing compiler required member 'System.Object.ToString'
-            //     static string M1(string s, char c) => s + c;
-            Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "c").WithArguments("System.Object", "ToString").WithLocation(13, 47),
-            // (14,43): error CS0656: Missing compiler required member 'System.Object.ToString'
-            //     static string M2(string s, char c) => c + s;
-            Diagnostic(ErrorCode.ERR_MissingPredefinedMember, "c").WithArguments("System.Object", "ToString").WithLocation(14, 43));
+        // We don't use object.ToString() or char.ToString() in the final codegen.
+        var verifier = CompileAndVerify(comp, expectedOutput: ExecutionConditionUtil.IsCoreClr ? "sccs" : null, verify: Verification.FailsPEVerify);
```

Comment:
For these missing `ToString` tests, it didn't seem important to me to preserve the existing behavior. The existing behavior was simply a factor of the incremental build-up process of the local rewriter, and while I do think it is good to verify our behavior with non-compliant standard libraries, I don't think it matters that we ensure an error is produced here. If we break it again in some future refactoring, oh well.